### ISSUE TITLE
Docs: How to run the Todo app tutorial with sqlite?

### ISSUE THREAD

**Issue by orgarten** [litestar-org/litestar#2468]
### Summary

I am currently trying to get started with litestar with the [todo app with sqlalchemny tutorial](https://docs.litestar.dev/2/tutorials/sqlalchemy/0-introduction.html). 

Running the code provided by this page does not run currently because the table `todo_items` is missing in the database.

```
(sqlite3.OperationalError) no such table: todo_items [SQL: SELECT todo_items.title, todo_items.done FROM todo_items] (Background on this error at: https://sqlalche.me/e/20/e3q8)
```

This is triggered by `result = await session.execute(query)` in the function `get_todo_list`.

It does make sense to me, because no table is visibly created. I am not very familiar with SQLalchemy and wasn't able to figure out what to do. Not sure if this is a bug in the code or if it's just a missing hint in the docs that one has to create the table before.

---
> [!NOTE]  
> While we are open for sponsoring on [GitHub Sponsors](https://github.com/sponsors/litestar-org/) and 
> [OpenCollective](https://opencollective.com/litestar), we also utilize [Polar.sh](https://polar.sh/) to engage in pledge-based sponsorship.
>
> Check out all issues funded or available for funding [on our Polar.sh Litestar dashboard](https://polar.sh/litestar-org)
> * If you would like to see an issue prioritized, make a pledge towards it!
> * We receive the pledge once the issue is completed & verified
> * This, along with engagement in the community, helps us know which features are a priority to our users.

<a href="https://polar.sh/litestar-org/litestar/issues/2468">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/litestar-org/litestar/issues/2468/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/litestar-org/litestar/issues/2468/pledge.svg">
</picture>
</a>

**Comment by peterschutt** [comment]
Thanks, a contribution to add this would be very welcome!

**Comment by orgarten** [comment]
I will add this within the next couple of days. 

When I build the docs with `make docs-serve` I don't see different code for different python versions. Where is that configured?

**Comment by orgarten** [comment]
Closed by merge of #2497
